src/symbolize/symbolizer.rs

```diff
@@ -171,6 +184,18 @@ fn default_apk_dispatcher(info: ApkMemberInfo<'_>, debug_syms: bool) -> Result<B
 }
 
 
+/// Information about an address space member of a process.
+#[derive(Clone, Debug)]
+pub struct ProcessMemberInfo<'dat> {
+    /// The "pathname" component in a `/proc/[pid]/maps` entry. See
+    /// `proc(5)` section `/proc/[pid]/maps`.
+    pub member_entry: &'dat PathName,
```

Comment:
similar question as for APK, why not provide the original address that we mapped to this "member"? I'd argue that you should also provide matched memory range, because it can be easily used through `/proc/[pid]/map_files/<range>`, which seems convenient for user.

Reply:
I agree that the memory range makes sense to include. Will add it.

> why not provide the original address that we mapped to this "member"?

What do you mean by "original address"?

Reply:
re: original address. It's the address (presumably from captured stack trace) that caused blazesym to request information for this ProcessMemberInfo (so the absolutely address within this process). As with above, I don't have specific use case, but it might be useful for logging/debugging and for completeness. But again, it's probably minor (and can be added later if necessary)